admin-events.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Document</title>
+
+</head>
+<header><h1><p></p>Incremental Game Framework</p></h1></header>
+<body>
+        <nav>
+            <a href="file:///Users/erickcilia/Desktop/cs-3220-spring-2018-erck76/admin-info.html">Game Information</a> |
+            <a href="file:///Users/erickcilia/Desktop/cs-3220-spring-2018-erck76/admin-generators.html">Generators</a> |
+            <a href="file:///Users/erickcilia/Desktop/cs-3220-spring-2018-erck76/admin-events.html">Events</a>
```

Comment:
absolute path should be avoided.

Reply:
especially those with `file://` protocol, it's not going to work outside of your machine.